### PR TITLE
Move sttr files to environment specific folders

### PR DESCRIPTION
**Pull request by afjlambert**
This is work in advance of autofill.

Benefits:
- For easy debugging / development
- No change of accidentally pushing staging sttr configs to production
- Got rid of yargs, security issue
- No more staging sttr-config in repo (outcomes.json)
- `git diff` doesn't complain as much when you are running with non-prod sttr config

Hope you like it 😃

### CODE REVIEW

File: apps/client/scripts/sttr_build/index.js [hunk 1]
```diff
@@ -62,10 +57,10 @@ function checkStatus(res) {
 }
 
 (async () => {
-  const permitIds = await fetch(listUrl, { headers })
+  fetch(listUrl, { headers })
```
Comment (robinpiets) [inline apps/client/scripts/sttr_build/index.js:60]: Do you need `async` when you remove `await`?

File: apps/client/scripts/sttr_build/index.js [hunk 2]
```diff
@@ -1,29 +1,22 @@
...
 const MAX_PARALLEL = 6;
 const env = process.env.STTR_ENV === "production" ? "PROD" : "STAGING";
+const outputDir = path.join(
+  __dirname,
+  "..",
+  "..",
+  "public",
+  "sttr",
+  env.toLowerCase()
+);
```
Comment (robinpiets) [inline apps/client/scripts/sttr_build/index.js:19]: Nice!

File: apps/client/.gitignore [hunk 3]
```diff
@@ -1,5 +1,7 @@
-public/sttr/*.xml
-public/sttr/*.source.json
+public/sttr/staging
```
Comment (robinpiets) [inline apps/client/.gitignore:1]: So we're never commiting the staging files.. makes sense

File: apps/client/scripts/sttr_build/index.js (hunk 1 above)
Comment (afjlambert) [reply apps/client/scripts/sttr_build/index.js:60]: Yes there is an await few lines below.

**Review by robinpiets — APPROVED** [review]

**Comment by robinpiets** [comment]
ready to merge!

**Pull request merged**